src/SystemLog/Logger.cs

```diff
@@ -14,7 +14,7 @@ protected override void Configure()
             Settings.Load(GetConfiguration());
         }
 
-        void ILogPlugin.Log(string source, LogLevel level, string message)
+        void ILogPlugin.Log(string source, LogLevel level, object message)
```

Comment:
Need to consider whether the type of `message` is `Exception` or other.

Reply:
What should be the behaviour when it's an Exception?

Reply:
https://github.com/neo-project/neo-node/commit/82033b46d36a69b31e24e6f421e5e174a2ca0d88#diff-2ad153cafa6b784ceb93f0f6a668bcbdL31-L49